gallery/visualtests/testUtils.js

```diff
@@ -14,7 +14,11 @@ const pageUrl = `file://${__dirname}/../dist/index.html`;
 
 const { AxePuppeteer } = require('@axe-core/puppeteer');
 
+const TOOLTIP_WAIT = 800;
```

Comment:
Note that prior code for waiting for tooltips tended to use 500ms for the wait.  I found that this wasn't quite adequate as some tooltips weren't done appearing in some snapshots.